app/views/lunch/_restaurants.html.erb

```diff
@@ -2,7 +2,7 @@
   <h3><%= restaurant[:name] %></h3>
   <dl>
     <% restaurant[:meals].each do |meal| %>
-    <dt><%= meal[:title] %></dt>
+    <dt><%= meal[:title] %><span class=allergens><%= meal[:allergens] %></span></dt>
```

Comment:
HTML attributes should be enclosed in quotes: `class="allergens"`

Here is the place for formatting the allergens array! 
Create a helper function (in `lunch_helper.rb`) for the more advanced logic of choosing between printing `nil`, empty array, or join with `,`.